<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseSetting</class>
 <widget class="QDialog" name="DatabaseSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paramètre base de données</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Information sur la base de données</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="addressLabel">
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="addressLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="identifiantLabel">
         <property name="text">
          <string>Identifiant</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="identifiantLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>Mot de passe</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="portSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65565</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="validButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Fermer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>DatabaseSetting</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addressLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DatabaseSetting</receiver>
   <slot>formChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>identifiantLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DatabaseSetting</receiver>
   <slot>formChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DatabaseSetting</receiver>
   <slot>formChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>portSpinBox</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>DatabaseSetting</receiver>
   <slot>formChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>validButton</sender>
   <signal>clicked()</signal>
   <receiver>DatabaseSetting</receiver>
   <slot>valid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>formChange()</slot>
  <slot>valid()</slot>
 </slots>
</ui>
